docker-compose/tailscale/Caddyfile

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
Is this caddyfile something that tailscale needs?

Reply:
There is a readme file next to it (actually two readme files).

This is an example how one could get this up and running with a Caddy Reverse Proxy, which is responsible for the Tailscale connection. Caddy needs a Caddyfile.

I can also remove this entirely and put it into a gist or a blog post or something

Reply:
I removed it.